### DISCUSSION TITLE
How to download on a macbook?

### DISCUSSION THREAD

**Discussion by mattiestreams2** [w-okada/voice-changer#733] [Q&A]
I need help downloading on a macbook

**Comment by FernThatCodes** [comment]
If you are on a m1 MacBook, download the version "MMVCServerSIO_mac_onnxcpu-nocuda_v.1.5.3.12.zip" on this website (https://huggingface.co/wok000/vcclient000/tree/main) After, double click on the zip file to unzip it, open the folder, run the file that says "startHttp.command" this should open up your terminal and after it's done loading, the voice changer gui should appear. If not try posting a bug report in the issues section or follow this tutorial (https://youtu.be/_JXbvSTGPoo) hope that works! Good luck :)

**Comment by namen-victor** [comment]
Does this work on Mckbook Pro 2019 15 inches? .. it has 
Radeon Pro 555X 4 GB
Intel UHD Graphics 630 1536 MB

**Comment by FernThatCodes** [comment]
Unfortunately, there is no current support for intel Macs right now

**Reply by namen-victor** [reply]
Thanks

**Comment by SaiStarz03** [comment]
I have setup everything and the guy opens but my voice doesn't change no matter what I do
